### PR TITLE
create target dir explicitly

### PR DESCRIPTION
**Pull request by pjfanning**
Try to fix up the logic removed in https://github.com/pjfanning/sbt-source-dist/pull/6#discussion_r1095639706

### CODE REVIEW

**Comment by pjfanning** [comment]
@mdedetrich this is my suggestion as to how to ensure the target/dist dir is there

**Review by mdedetrich — APPROVED** [review]
Lgtm

**Review by mdedetrich — CHANGES_REQUESTED** [review]
Actually now that I think of it, should you check that `dist` doesn't exist before creating it?

File: src/main/scala/com/github/pjfanning/sourcedist/SourceDistGenerate.scala [hunk 1]
```diff
@@ -29,25 +29,26 @@ private[sourcedist] object SourceDistGenerate {
     val dateTimeFormatter = DateTimeFormatter.BASIC_ISO_DATE
     val dateString = LocalDate.now().format(dateTimeFormatter)
     val baseFileName = s"$prefix-src-$version-$dateString"
-    val toZipFileName = new File(s"$targetDir/$baseFileName.zip")
-    val toTgzFileName = new File(s"$targetDir/$baseFileName.tgz")
+    IO.createDirectory(new File(targetDir))
```
Comment (mdedetrich) [inline src/main/scala/com/github/pjfanning/sourcedist/SourceDistGenerate.scala]: We should check that the directory doesn't already exist before trying to create it.

**Comment by pjfanning** [comment]
@mdedetrich I added a check for if the dir rexists and a local run with incubator-pekko seems to be ok

**Review by mdedetrich — APPROVED** [review]
Lgtm

**Pull request merged**